packages/plugin-e2e/tests/as-viewer-user/permissions.spec.ts

```diff
@@ -2,7 +2,6 @@ import { expect, test } from '../../src';
 
 test('should redirect to start page when permissions to navigate to page is missing', async ({ page }) => {
   await page.goto('/');
-  const homePageTitle = await page.title();
   await page.goto('/datasources', { waitUntil: 'networkidle' });
-  expect(await page.title()).toEqual(homePageTitle);
+  expect(page.url()).not.toContain('datasources');
```

Comment:
This and the next test started failing as a consequence of scenes powered dashboards. This is a simple change to cater for that.